modules/core/src/arithm.simd.hpp

```diff
@@ -1593,7 +1593,7 @@ struct op_div_f
 {
     static inline Tvec r(const Tvec& a, const Tvec& b)
     {
-        const Tvec v_zero = Tvec();
+        const Tvec v_zero = vx_setall<typename Tvec::lane_type>(0);
```

Comment:
IMO it make sense to add vx_setzero template as well and use it here instead

Reply:
Will be added as a separate PR